test/indexer/test_hca_indexer.py

```diff
@@ -1251,7 +1251,17 @@ def test_accessions_fields(self):
         for hit in hits:
             contents = hit['_source']['contents']
             project = one(contents['projects'])
-            self.assertEqual(['SRP000000'], project['insdc_project_accessions'])
+            accessions = [
+                {'namespace': 'array_express', 'accession': 'E-AAAA-00'},
+                {'namespace': 'geo_series', 'accession': 'GSE00000'},
+                {'namespace': 'insdc_project', 'accession': 'SRP000000'},
+                {'namespace': 'insdc_project', 'accession': 'SRP000001'},
+                {'namespace': 'insdc_study', 'accession': 'PRJNA000000'}
+            ]
+            entity_type, aggregate = self._parse_index_name(hit)
+            if entity_type == 'project':
+                self.assertEqual(accessions, project['accessions'])
```

Comment:
As mentioned before, your additions should follow existing code, whenever possible.